components/image/image.css

```diff
@@ -0,0 +1,15 @@
+.image-square {
+  width: 100px;
```

Comment:
suggestion: you can also use relative units % or rem for more responsive

Reply:
I wanted the square and rounded images to be fixed and so the fixed width, while I've used the styles for responsive images different.

Reply:
ok fine